roles/webmail/templates/var_www_roundcube_config_config.inc.j2

```diff
@@ -25,7 +25,7 @@ $config = array();
 // For examples see http://pear.php.net/manual/en/package.database.mdb2.intro-dsn.php
 // NOTE: for SQLite use absolute path (Linux): 'sqlite:////full/path/to/sqlite.db?mode=0646'
 //       or (Windows): 'sqlite:///C:/full/path/to/sqlite.db'
-$config['db_dsnw'] = 'mysql://roundcube:pass@localhost/roundcubemail';
+$config['db_dsnw'] = 'sqlite:////var/www/roundcube/db/sqlite.db?mode=0664';
```

Comment:
Locating the database file here means it doesn't get backed up.  This should be moved to a location such as `/decrypted/roundcube/sqlite.db`.